nautobot/core/templatetags/helpers.py

```diff
@@ -798,6 +798,19 @@ def render_job_run_link(value):
     return str(value)
 
 
+@library.filter()
+@register.filter()
+def label_list(value, suffix=""):
+    """Render a list of values with optional suffix (like 'MHz') as span labels."""
+    if not value:
+        return ""
+    return format_html_join(
+        " ",
+        '<span class="label label-default">{0}{1}</span>',
+        ((item, f" {suffix}" if suffix else "") for item in value),
```

Comment:
Seems wrong to me to automatically prepend `" "` to the provided suffix - shouldn't the suffix itself have control over that?